package/common/cpp/react/renderer/components/RNCSlider/RNCSliderMeasurementsManager.h

```diff
@@ -1,4 +1,6 @@
 #ifdef ANDROID
+#pragma once
```

Comment:
Ah, yes! That's mine mistake - I got confused `#ifdef` with `ifndef` and suggested removing `once` during review of new arch measurement PR #589. 🙇‍♂️